lib/rules/max-line-length/index.js

```diff
@@ -34,7 +34,11 @@ const rule = function (maxLength, options) {
 
     // Collapse all urls into something nice and short,
     // so they do not throw the game
-    const rootString = root.toString().replace(/url\(.*\)/ig, "url()")
+    const rootString = root.toString()
+    // Replace `url(something)` with `url()`
+    .replace(/url\(.*\)/ig, "url()")
+    // Replace `@import "something"` with `@import ""`
+    .replace(/\@import\s+['"].*['"]/ig, "@import ")
```

Comment:
Ooooo, I think my misunderstanding was in forgetting that the `*` is *non-greedy by default*, so the last `['"]` will match the last possible quotation mark instead of first one it finds. You're right: carry on!